index.html

```diff
@@ -1,168 +1,161 @@
+                <figcaption>Welcome to my page!</figcaption>
+            </section>
+
+            <section id="contact" class="contact">
+                <h3>CONTACT</h3>
+                    <div class="contact-item">
+                        <span class="headline">Mobile:</span><br>
+                        <img src="images/telephone.png" alt="icon telephone" class="icon"> 0176/41518761<br>
+                    </div>
+                    <div class="contact-item">
+                        <span class="headline">Email:</span><br>
+                        <img src="images/email.png" alt="icon email" class="icon"> anita.s1@gmx.de<br>
+                    </div>
+                    <div class="contact-item">
+                        <span class="headline">Location:</span><br>
+                        <img src="images/home.png" alt="icon location" class="icon"> Munich</p>
+                    </div>
+            </section>
+            
+            <section class="biography">
+                <h3>BIOGRAPHY</h3>
+                <p>Lorem, ipsum dolor sit amet consectetur adipisicing elit. Voluptatem corrupti enim tempora maiores nisi dicta. Eos unde est dolor officia illo alias repudiandae harum corporis error iure recusandae, facere at.
+                    Sed eveniet molestias doloribus, incidunt explicabo nihil est voluptates ea voluptas natus quisquam et dolorum, officiis omnis temporibus excepturi accusantium. Ipsa quisquam porro ipsum placeat provident quibusdam officiis ex autem?
+                    Quam reiciendis obcaecati qui, sint, sequi culpa, at assumenda voluptatem distinctio autem doloribus enim aliquid nobis earum ea perferendis pariatur? Quae itaque cumque illum maxime explicabo nam aliquam earum sapiente!
+                    Qui accusantium sit nulla nobis quae laudantium tempora ad dolor culpa possimus, voluptatum sint sequi officia eum quas minima laboriosam molestiae animi rem iste. Atque quibusdam itaque praesentium possimus cum.
+                   </p>
+            </section>
+
+            <section class="skills">
+                <h3>SKILLS</h3>
+                    <ul>
+                        <li>Git/Github</li>
+                        <li>HTML</li>
+                        <li>CSS</li>
+
+                    </ul>
+            </section>
+
+            <section class="hobbies">
+                <h3>HOBBIES</h3>
+                    <ul>
+                        <li>Travelling</li>
+                        <li>Drawing</li>
+                        <li>Hiking</li>
+                    </ul>
+            </section>
+
+        </section> 
+    
+        <section id="cv" class="cv">
+            <h2 class="headline-cv">CV</h2>
+
+            <section class="professional-experience">
+                <h3>PROFESSIONAL EXPERIENCE</h3>
+
+                <div class="stations">
+                    <div>
+                        <p><strong>Junior Marketing Manager</strong><br>
```

Comment:
```suggestion
                        <p><strong>Junior Marketing Manager</strong>
```
Try to avoid `br`tags and rather add new lines via CSS